roles/confluent.control_center/tasks/main.yml

```diff
@@ -54,6 +58,19 @@
     group: "{{control_center_group}}"
     recurse: true
 
+# Archive File deployments need to create SystemD service units
+- name: Test for Control Center Service
+  stat:
+    path: "{{control_center.systemd_file}}"
+  register: control_center_service_systemd
+
+- name: Create Control Center Service
```

Comment:
template module has a force option that does the if exists logic

Reply:
Updated this section to not use `stat` and use the `force` property of the `template` task in commit bdfe4610